sfm_pc/management/commands/download_data.py

```diff
@@ -0,0 +1,222 @@
+import os
+import csv
+import io
+
+from oauth2client.service_account import ServiceAccountCredentials
+from apiclient.discovery import build
+from googleapiclient.http import MediaIoBaseDownload
+import httplib2
+
+from tqdm import tqdm
+
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    help = 'Download entity, source, and location data from Google'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--sources_doc_id',
+            dest='sources_doc_id',
+            default='1o-K13Od1pGc7FOQ-JS8LJW9Angk3_UPZHTmOU7hD3wU',
+            help='Import data from specified Google Sheets Document'
+        )
+        
+        parser.add_argument(
+            '--location_doc_id',
+            dest='location_doc_id',
+            help='Import location data from specified Google Drive file'
+        )
+
+        parser.add_argument(
+            '--entity_doc_id',
+            dest='entity_doc_id',
+            default='1o-K13Od1pGc7FOQ-JS8LJW9Angk3_UPZHTmOU7hD3wU',
+            help='Import data from specified Google Sheets Document'
+        )
+        
+        parser.add_argument(
+            '--country_code',
+            dest='country_code',
+            help='Country code for the import'
+        )
+
+    def handle(self, *args, **kwargs):
+        entity_doc_id = kwargs['entity_doc_id']
+        location_doc_id = kwargs['location_doc_id']
+        sources_doc_id = kwargs['sources_doc_id']
+        country_code = kwargs['country_code'].rstrip()
```

Comment:
I agree that passing the arguments around can be a bit confusing. I think the latest version of this code makes more sense without setting them as class attributes, because the `_create_csv_files` caller is more generic with the `doc_id` (and so are the functions within that function that use that parameter). With the latest change, I don't know if it's necessary anymore to add them as attributes. Take a look and let me know if you agree?

Aside, I thought about adding them as attributes originally, but decided to pass as arguments because I understand class attributes to be more useful for state-like objects, where passing arguments is more pure function-like which makes it easier to understand the input for a function rather than having to grok the entire class. I sometimes find class attributes confusing. They can feel like they *come from above* and code makes more sense to me when arguments are passed directly into the function call, though admittedly it can get confusing when you pass the same argument to multiple levels of function callers (in which case that it might be better to add as an attribute). This isn't based on any principle, just my own style hunch and impression from code examples I've seen. There's a philosophical debate over "is this class a thing with these attributes" but this is work, not play!